### DISCUSSION TITLE
gcc5.3.0 and nlohmann::json

### DISCUSSION THREAD

**Discussion by sreerajchundayil** [nlohmann/json#3018] [Q&A] [answered]
Hello Everyone,

  We are currently using nlohmann json with gcc620 and above successfully.
  Now we are planning to use nlohmann json with gcc530.
  Since C++11 feature is in experimental mode in this gcc version, has any one faced any issue in using this library with it?
  Or can we confidently go ahead with using nlohmann json in gcc530 by enabling -std=c++11 switch?

**Comment by nlohmann** [comment, marked as answer]
We are aware of some issues in GCC 4.8 (https://github.com/nlohmann/json#supported-compilers), but successfully compile and test the library on it. We also build with GCC 5.4.0 (Ubuntu 5.4.0-6ubuntu1~16.04.12) in the CI. That said, I would not expect any problems with GCC 5.3.0.

If you're facing issues anyhow, please open an issue for this!
